data/processors/maps.py

```diff
@@ -225,21 +241,23 @@ def assign_roomfinder_maps(data):
 def build_roomfinder_maps(data):
     """ Generate the map information for the Roomfinder maps. """
     
-    # Read the Roomfinder maps
+    # Read the Roomfinder and custom maps
     with open("external/maps_roomfinder.json") as f:
         maps_list = json.load(f)
+    custom_maps = _load_custom_maps()
     
-    # For each map, we calculate the boundaries in UTM, because 
+    # For each map, we calculate the boundaries in UTM beforehand
     maps = {}
-    for m in maps_list:
+    for m in maps_list + list(custom_maps.values()):
```

Comment:
`_load_custom_maps()` should return the format we need up here. 
Thus I think we should move this conversion into the more proper method.